vaccine_feed_ingest/runners/ca/metrolink/normalize.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+_source_name = "ca:metrolink"
+
+
+def _get_id(site: dict) -> str:
+    name = _get_name(site)
+    city = _get_city(site)
+    return f"{name}:{city}"
+
+
+def _get_name(site: dict) -> str:
+    return site["name"]
+
+
+def _get_city(site: dict) -> str:
+    return site["city"]
+
+
+def _get_address(site: dict):
+    return schema.Address(
+        street1=site["address"],
+        city=_get_city(site),
+        zip="0",
+        state="CA",
+    )
+
+
+def _get_notes(site: dict):
+    ret = []
+    ret.append("metrolink_line__ca_specific:" + site["metrolink_line"])
+    ret.append("metrolink_station__ca_specific:" + site["metrolink_station"])
```

Comment:
The primary users of notes will be humans reading the information. Could we update this to be sentence-formatted, like "Near the <> station on the <> metrolink line."?

Reply:
Good call, thanks. Done.